### PR TITLE
Fixed issue 9953: linsolve([ ], ...) now returns S.EmptySet

### PR DESCRIPTION
**Pull request by devyanikota**
ping @leosartaj @aktech

### CODE REVIEW

**Comment by leosartaj** [comment]
You should also write a test case. You can add a testcase at the end of [this](https://github.com/sympy/sympy/blob/master/sympy/solvers/tests/test_solveset.py) file. Something like this.

```
def test_issue_9953():
  assert linsolve([ ], x) == S.EmptySet
```

File: sympy/solvers/tests/test_solveset.py [hunk 1]
```diff
@@ -1013,3 +1013,6 @@ def test_issue_failing_pow():
 
 def test_issue_9849():
     assert solveset(Abs(sin(x)) + 1, x, S.Reals) == S.EmptySet
+
```
Comment (leosartaj) [inline sympy/solvers/tests/test_solveset.py]: One more blank line.

**Comment by leosartaj** [comment]
Maybe you can add an example in the docstring regarding the new behavior.

**Comment by leosartaj** [comment]
Other than my few comments. This looks good. +1 to merge, if tests pass.

**Comment by aktech** [comment]
+1

File: sympy/solvers/tests/test_solveset.py [hunk 2]
```diff
@@ -1013,3 +1013,8 @@ def test_issue_failing_pow():
...
+
+def test_issue_9953():
+    assert linsolve([ ], x) == S.EmptySet
+
```
Comment (leosartaj) [inline sympy/solvers/tests/test_solveset.py]: Please remove this line.

File: sympy/solvers/solveset.py [hunk 3]
```diff
@@ -1194,8 +1194,15 @@ def linsolve(system, *symbols):
     >>> linsolve(system, x, y)
     {(x, y)}
 
+    * For an empty system linsolve returns empty set
```
Comment (leosartaj) [inline sympy/solvers/solveset.py]: Add a new line.

**Comment by leosartaj** [comment]
Tests pass. Merging.

**Comment by leosartaj** [comment]
Thanks for you work @Devyani-Divs.

**Comment by Devyani-Divs** [comment]
Ah, finally ! Worth the wait :) thanks @leosartaj for helping me out.

**Comment by leosartaj** [comment]
You're Welcome.

**Pull request merged**